### ISSUE TITLE
DB error when receiving a private message from Diaspora

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#7196]
Apparently when we receive a private message from Diaspora, we try to cram a string (a `mail.parent-uri` in an integer field `notify.parent` and it gives this error:
```
[ERROR]: DB Error {
  "code": 1366,
  "error": "Incorrect integer value: 'mrpetovan@diaspora.permutationsofchaos.com:74a1e680ee160136dc4e6f2d45436a48' for column `friendica`.`notify`.`parent` at row 1",
  "callstack": "DBA::insert, notification, Mail::insert, Diaspora::receiveMessage, Diaspora::dispatch, receive_post, LegacyModule::runModuleFunction, LegacyModule::post",
  "params": "INSERT INTO `notify` (`hash`, `name`, `url`, `photo`, `date`, `uid`, `link`, `iid`, `parent`, `type`, `verb`, `otype`, `name_cache`) VALUES ('d87d60e6be83c02d84775d50293cd99e9d29d40d568c66681cf4cea1ca570677', 'Hypolite Petovan', 'https:\\/\\/diaspora.permutationsofchaos.com\\/u\\/mrpetovan', 'https:\\/\\/diaspora.permutationsofchaos.com\\/uploads\\/images\\/thumb_large_a8a1a4815a0baea1ec4b.jpg', '2019-05-27 02:11:08', 1, 'https:\\/\\/friendica.mrpetovan.com\\/message\\/977', '977', 'mrpetovan@diaspora.permutationsofchaos.com:74a1e680ee160136dc4e6f2d45436a48', 16, 'http:\\/\\/activitystrea.ms\\/schema\\/1.0\\/post', 'mail', 'Hypolite Petovan')"
} - {"file":"DBA.php","line":697,"function":"e","uid":"0bfe59","process_id":16247}
```
The most puzzling is that it doesn't seem to affect anything? I'm still getting notifications of new mail, but maybe I'm not getting it in the right spot?

I'm not sure either how to fix this error:
- Either we coerce the parent-uri into an integer (`convid`?),
- Or to change the `notify.parent` field to a string.

Help me @annando , you're my only hope.

**Comment by nupplaphil** [comment]
Fixed with #7196

**Comment by AlfredSK** [comment]
No. Fixed with #7201 :-)
